docker_push.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+#Script used to push new docker images to docker hug by travis.
+
+./gradlew -PcloudType=local :distributions:demo-server:dockerize
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
can you document how these env variables are set?

Reply:
Done.